ztp/siteconfig-generator/siteConfig/siteConfigBuilder_test.go

```diff
@@ -582,6 +588,11 @@ func Test_ExtraManifestSearchPath(t *testing.T) {
 		mapSourceCR := cr.(map[string]interface{})
 
 		if mapSourceCR["kind"] == "ConfigMap" {
+			configMapMetadata := mapSourceCR["metadata"].(map[string]interface{})
```

Comment:
Not sure, why do we need to add this , this siteConfig is v1 so it suppose to not have the siteConfigMap. Do I'm missing something here ?

Reply:
You are right, they are not needed. I think they were here since before I rebased with the SiteConfig V2.